ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDFFromUnixTime.java

```diff
@@ -60,7 +62,7 @@
 
   private transient SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
Remove this variable if not being used anywhere?

Reply:
Removed the variable from the code.